modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -235,8 +235,8 @@ def update_status!(status:, code: nil, detail: nil)
 
       email_handler = Events::Handler.new(event_type: :sc_received, opts: {
                                             email_identifier: email_identifier,
-                                            first_name: veteran_first_name,
-                                            date_submitted: veterans_local_time.iso8601,
+                                            first_name: signing_appellant.first_name,
```

Comment:
We specifically want the veteran's first name here, not the signing appellant - the claimant email template says something along the lines of "hi <claimant name> - thanks for your submission of appeal on <veteran's first name> behalf" which would be weird if it said the claimant's name twice :)

Reply:
Yes, I see that now lol